gum/backend-x86/gumspinlock-amd64-msc.c

```diff
@@ -9,6 +9,8 @@
 #include "gumx86writer.h"
 #include "gummemory.h"
 
+#include <capstone.h>
+
```

Comment:
This include should not be necessary, as gumx86writer.h should provide it due to the instruction type being part of its API.